htdocs/accountancy/bookkeeping/list.php

```diff
@@ -315,6 +314,7 @@
 
 print '<form method="GET" id="searchFormList" action="' . $_SERVER["PHP_SELF"] . '">';
 print '<div class="tabsAction">' . "\n";
+print '<div class="inline-block divButAction"><a class="butAction" href="./listbyaccount.php">' . $langs->trans("Bookkeeping") . ' ' . strtolower($langs->trans("By")) . ' ' . strtolower($langs->trans("AccountAccounting")) . '</a></div>';
```

Comment:
you can add a key language in the language file rather than to compile 3 words with a strtolower